### DISCUSSION TITLE
Games not showing up when in simple menu

### DISCUSSION THREAD

**Discussion by Ltnetsudo** [eduardofilo/RG350_adam_image#127] [General]
Hello,  I am following the guide and after loading Roms they are not showing up on the simplemenu.m app. I can only see them if I back out of simplemenu and manually load the respective emulator. Any ideas why this might be happening? The Roms run fine when they are manually loaded. Any ideas?

**Comment by eduardofilo** [comment]
Did you press `B` two times to go down to the group selection?

**Reply by eduardofilo** [reply]
Check the new FAQ number 10 to see if it helps you.

https://github.com/eduardofilo/RG350_adam_image#10-i-have-loaded-roms-on-the-external-card-but-they-do-not-show-up-in-simplemenu
